src/Service/Configurations/RuntimeConfigValidator.cs

```diff
@@ -36,13 +36,13 @@ public class RuntimeConfigValidator : IConfigValidator
         // The claimType is invalid if there is a match found.
         private static readonly Regex _invalidClaimCharsRgx = new(_invalidClaimChars, RegexOptions.Compiled);
 
-        // "Reserved characters as defined in RFC3986 are not allowed to be present in the
-        // REST/GraphQL custom path because they are not acceptable to be present in URIs.
-        // " Refer here: https://www.rfc-editor.org/rfc/rfc3986#page-12.
-        private static readonly string _invalidPathChars = @"[\.:\?#/\[\]@!$&'()\*\+,;=]+";
+        // Reserved characters as defined in RFC3986 are not allowed to be present in the
+        // REST/GraphQL URI path because they are not acceptable to be present in URIs.
+        // Refer here: https://www.rfc-editor.org/rfc/rfc3986#page-12.
+        private static readonly string _invalidURIComponentChars = @"[\.:\?#/\[\]@!$&'()\*\+,;=]+";
```

Comment:
```suggestion
        private static readonly string _invalidUriCharacters = @"[\.:\?#/\[\]@!$&'()\*\+,;=]+";
```

Since these reserved characters can't be included in URI's, this regex would be appropriate for any URI checking?

Reply:
Yeah